sdk/core/core-rest-pipeline/src/util/stream.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Readable } from "stream";
+import { BlobLike } from "../interfaces";
+import {
+  isInMemoryBlob,
+  isNodeReadableStream,
+  isStreamableBlob,
+  isWebReadableStream,
+} from "./typeGuards";
+
+function nodeStreamFromWebStream(webStream: ReadableStream): NodeJS.ReadableStream {
+  return Readable.from(
```

Comment:
Why aren't we using `fromWeb` and `toWeb`? https://nodejs.org/api/stream.html#streamreadabletowebstreamreadable-options

Reply:
Added a fill for @@asyncIterator so we can do it for now.